tempesta_fw/http.c

```diff
@@ -2259,6 +2259,7 @@ tfw_http_resp_fwd(TfwHttpResp *resp)
 	}
 	if (!req_retent) {
 		spin_unlock_bh(&cli_conn->seq_qlock);
+		tfw_http_resp_pair_free(resp->req);
```

Comment:
Are you convinced that this is the right thing to do?

Here's my understanding. `req_retent` is not set when there're no consecutive responses (from the start of `seq_queue`) that are ready to be sent out to a client. That's a perfectly valid case. The current `resp` has been marked as ready and it's waiting for its turn until the 'slots' before it are filled. Freeing it (and the paired request) would be incorrect. Moreover, if there's need to delete req/resp pair for some reason, the request needs to be removed from  `seq_queue` first.

Reply:
Yeah, that was just wrong mess in the patch. Thanks for noticing.